### ISSUE TITLE
monifu for scalajs 0.6.0

### ISSUE THREAD

**Issue by weihsiu** [monifu/monifu#37]
now that scalajs 0.6.0 is about to be released (it is on maven central already).  when can we expect an updated version that works with 0.6.0?

**Comment by alcarvalho** [comment]
I'm also interested in this. But from what I've seen, all the jasmine tests will need to be migrated to other framework. Probably [MiniTest](/monifu/minitest)?

**Comment by alexandru** [comment]
@weihsiu @alcarvalho - I published version `0.15.0` on Maven Central.

I haven't migrated the Jasmine tests, I just dropped them. This is because there's another branch I'm working on with a shared test suite, in order to achieve good test coverage for both the JVM and JS. That branch (towards-1.0) also breaks the API. But it's totally unstable right now - will be released soon.

In order to use version `0.15.0` the build settings have been changed, so now you can do:

```scala
libraryDependencies += "org.monifu" %%% "monifu" % "0.15.0"
```

Thanks for your patience. I've been super busy at work - but now I'm back full steam in getting a 1.0 out the door.

Cheers,
